syfosm-common-models/src/main/kotlin/no/nav/syfo/model/juridisk/JuridiskVurdering.kt

```diff
@@ -12,6 +13,7 @@ data class JuridiskVurdering(
     val juridiskHenvisning: JuridiskHenvisning,
     val sporing: Map<String, String>,
     val input: Map<String, Any>,
+    val tidsstempel: LocalDateTime,
```

Comment:
Jeg tror denne må være optional for i sikre bakoverkompatibilitet :)

Reply:
Det er sant